src/Data/MongoDBMock/contests.js

```diff
@@ -0,0 +1,163 @@
+const contests = [
+  {
+    contestId: 'a0347677-c3c9-4edc-9d46-fed4a958fdc2',
+    competitionName: 'XII Zawody im. Pana Starosty',
+    kynologiqueDepartment: 'Gdynia',
+    startDate: new Date(2022, 3, 22, 10, 0),
+    endDate: new Date(2022, 3, 23),
+    place: 'Gdańsk',
+    detailAdress: 'ul.Grunwaldzka 222',
+    judge: 'Alberto Makkaroni',
```

Comment:
my contestDetails component like these syntax:
`judges: ['Alberto Makkaroni']`
Can we change the name of property to `judges` and type of property `judges` to Array of strings?

Reply:
absolutely!